index.html

```diff
@@ -52,8 +53,8 @@ <h1>about</h1>
         <section>
             <article>
                 <h1 id="portfolio">portfolio</h1>
-                <a target="_blank" href="http://oratemate.com/">
-                    <img src="oratemate-01.png" class="image-icons">
+                <a class="image-icons" target="_blank" href="http://oratemate.com/">
+                    <img src="oratemate-01.png">
```

Comment:
might be worth adding alt="picture description" to img tags

Reply:
done